src/helpers/codetohtmlconverter.cpp

```diff
@@ -51,6 +53,7 @@ void CodeToHtmlConverter::initCodeLangs() Q_DECL_NOTHROW {
 }
 
 QString CodeToHtmlConverter::process(const QString &input) const {
+    qDebug() << "Going to highlight input: " << input.mid(0, 12) << ", with lang: " << _currentLang;
```

Comment:
You don't need leading and trailing spaces, qDebug seems to add them on its own.

Reply:
done

its only temporary though. Will remove these once the issue is identified.

Reply:
Ah, OK. Because the whitespaces are also in above debug log. 😁

Reply:
yeah, but does it really matter? (as said its temporary, just need it to confirm a few assumptions)